### PR TITLE
fix(envelope) Missing sdk info in header

### PR DESCRIPTION
**Pull request by krystofwoldrich**
## :loudspeaker: Type of change

- [x] Bugfix
- [ ] New feature
- [ ] Enhancement
- [x] Refactoring

## :scroll: Description

Outgoing envelope headers are containing correct sdk info.

## :bulb: Motivation and Context
Fix: https://github.com/getsentry/sentry-react-native/issues/2489

## :green_heart: How did you test it?

Unit tests, check the objects on the native layer

## :pencil: Checklist

- [x] I reviewed submitted code
- [x] I added tests to verify changes
- [x] All tests passing
- [x] No breaking changes

## :crystal_ball: Next steps
#skip-changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@
 - Add typings for app hang functionality ([#2479](https://github.com/getsentry/sentry-react-native/pull/2479))
 - Update warm/cold start span ops ([#2487](https://github.com/getsentry/sentry-react-native/pull/2487))
 - Detect hard crash the same as native sdks ([#2480](https://github.com/getsentry/sentry-react-native/pull/2480))
+- Add SDK Info to outgoing envelope header ([#2492](https://github.com/getsentry/sentry-react-native/pull/2492))
```
Comment (marandaneto) [inline CHANGELOG.md:14]: This is not an user-facing change, so I'd rather add `_#skip-changelog_`

File: src/js/client.ts [hunk 2]
```diff
@@ -15,8 +16,20 @@ import { Alert, LogBox, YellowBox } from 'react-native';
 import { ReactNativeClientOptions } from './options';
 import { NativeTransport } from './transports/native';
 import { createUserFeedbackEnvelope } from './utils/envelope';
+import { SDK_NAME, SDK_PACKAGE_NAME, SDK_VERSION } from './version';
 import { NATIVE } from './wrapper';
 
+const defaultSdkInfo: SdkInfo = {
```
Comment (marandaneto) [inline src/js/client.ts:22]: The `sdkinfo.ts` does that directly in the event, can we somehow unify this behavior? eg extracting and reusing the method.

Comment (krystofwoldrich) [reply src/js/client.ts:22]: yes

**Review by marandaneto — APPROVED** [review]
Thanks @krystofwoldrich

**Pull request merged**